### PR TITLE
Minor wording changes around RPM install

### PR DESCRIPTION
**Pull request by murdoaird**
Minor wording changes around RPM install

### CODE REVIEW

**Comment by drigodwin** [comment]
I think it might be better to have this as further tabs, like this [preliminary example](https://github.com/murdoaird/brooklyn-docs/pull/1).

**Comment by johnmccabe** [comment]
+1 to the RPM tab

File: guide/start/running.md [hunk 1]
```diff
@@ -9,7 +9,9 @@ This guide will walk you through deploying an example 3-tier web application to
 
 An overview of core [Brooklyn concepts](./concept-quickstart.html){:target="_blank"} is available for reference.
 
-Two methods of deployment are detailed in this tutorial, using virtualisation with Vagrant and an install in your own environment (such as your local machine or in your private/public cloud). Both assume that you are using Linux or Mac OS X. To get you up-and-running quickly, the Vagrant option will provision four compute nodes for you to deploy applications to. 
+Two methods of deployment are detailed in this tutorial, using virtualisation with Vagrant and an install in your own environment (such as your local machine or in your private/public cloud).
```
Comment (ahgittin) [inline guide/start/running.md]: [nit] [old-problem] US English is preferred

**Comment by ahgittin** [comment]
tabs are a neat idea, but this is a good improvement without them, merging

**Pull request merged**